standard/lexical-structure.md

```diff
@@ -1200,16 +1259,16 @@ The `#pragma` preprocessing directive is used to specify contextual information
 *end note*
 
 ```ANTLR
-Pp_Pragma
-    : Whitespace? '#' Whitespace? 'pragma' Pp_Pragma_Text
+PP_Pragma
+    : Whitespace? '#' Whitespace? 'pragma' PP_Pragma_Text
     ;
 
-Pp_Pragma_Text
+PP_Pragma_Text
     : New_Line
     | Whitespace Input_Character* New_Line
     ;
 ```
 
-The *Input_Character*s in the *Pp_Pragma-Text* are interpreted by the compiler in an implementation-defined manner. The information supplied in a `#pragma` directive shall not change program semantics. A `#pragma` directive shall only change compiler behavior that is outside the scope of this language specification. If the compiler cannot interpret the *Input_Character*s, the compiler can produce a warning; however, it shall not produce a compile-time error.
+The *Input_Character*s in the *PP_Pragma-Text* are interpreted by the compiler in an implementation-defined manner. The information supplied in a `#pragma` directive shall not change program semantics. A `#pragma` directive shall only change compiler behavior that is outside the scope of this language specification. If the compiler cannot interpret the *Input_Character*s, the compiler can produce a warning; however, it shall not produce a compile-time error.
```

Comment:
Should the - in *PP_Pragma-Text* be an underscore?

Reply:
I spotted this pre-existing typo when doing #342, I'll fix it here as well. There might be other fixes I found and made in #342, someone with good Git-fu I'm sure could merge them if we go with #339. I'm making all the fixes you found in both branches.